src/sagemaker_huggingface_inference_toolkit/transformers_utils.py

```diff
@@ -40,43 +39,40 @@ def is_aws_neuron_available():
     return _aws_neuron_available
 
 
+def strtobool(val):
```

Comment:
Shouldn't this return an actual boolean value `True` or `False` rather than `1` or `0`?

Reply:
Did you miss this comment?